docs/compute/auth/updateProfile.md

```diff
@@ -0,0 +1,46 @@
+# Update your Profile and Credentials
+
+After you have logged in, in the upper right-hand corner of the screen, you will see your name. Click on your name,
+then click on the menu item **Account settings**. The User settings page will open with profile tab selected, from 
+where you can change your first and last name. You cannot change your email address and will need to create a new 
+account if you wish to change your email address. You can also log out by clicking on the **Log out** menu item.
+
+  <figure>
+    <a  href="../../../assets/images/auth/profile.png">
+      <img src="../../../assets/images/auth/profile.png" alt="User profile" />
+    </a>
+    <figcaption>Update profile</figcaption>
+  </figure>
+
+## Update credential
+To update your credential go to the **Authentication** tab as shown in the image below.
+  <figure>
+    <a  href="../../../assets/images/auth/updatePassword.png">
+      <img src="../../../assets/images/auth/updatePassword.png" alt="Update credentials">
+    </a>
+    <figcaption>Update credential</figcaption>
+  </figure>
+
+## Enable/Reset Two-factor Authentication
+
+You can also change your two-factor setting in the **Authentication** tab. Second factor authentication
+is only available if it is enabled from the cluster administration page.
+
+  <figure>
+    <a  href="../../../assets/images/auth/2fa-enabled.png">
+      <img src="../../../assets/images/auth/2fa-enabled.png" alt="Two-factor Authentication">
+    </a>
+    <figcaption>Enable Two-factor Authentication</figcaption>
+  </figure>
+
+After enabling or resetting two-factor you will be presented with a QR Code. You will then need to scan the QR code 
+to add it on your phone's authenticator application. If you miss this step, you will have to recover your smartphone 
+credentials at a later stage. 
```

Comment:
Add a link to the documentation for recovering the second factor authentication.